runtimes/cloudformation/cmd/handler/main.go

```diff
@@ -78,6 +79,11 @@ func GetConfig() *cfnpatcher.Configuration {
 	disableRepoHints := os.Getenv("KILT_DISABLE_REPO_HINTS")
 	logGroup := os.Getenv("KILT_LOG_GROUP")
 	parameterizeEnvars := os.Getenv("KILT_PARAMETERIZE_ENVARS")
+	sidecarEssential := os.Getenv("KILT_SIDECAR_ESSENTIAL")
+	sidecarCpu := os.Getenv("KILT_SIDECAR_CPU")
+	sidecarMemoryLimit := os.Getenv("KILT_SIDECAR_MEMORY_LIMIT")
+	sidecarMemoryReservation := os.Getenv("KILT_SIDECAR_MEMORY_RESERVATION")
+	sidecarConfig := os.Getenv("KILT_SIDECAR_CONFIG")
```

Comment:
Are these documented anywhere? I know `KILT_LOG_GROUP` etc are documented in `kilt-definitions/cloudformation/serverless-patcher/README.md` already, should they be moved into a README in this repo?

Non-blocking comment.